### ISSUE TITLE
Unable to scrap certain coin symbols

### ISSUE THREAD

**Issue by bicced** [PyDataBlog/Coinsta#12]
Some symbols such as yoyow and gxc is not recognized.

coinsta.exceptions.WrongCoinCode: 'YOYOW' is unavailable on CoinMarketCap.com. Please check the website for the right ticker information code

**Comment by PyDataBlog** [comment]
@bicced it seems that there has been some new changes with these new coins. I will look into and make the necessary changes.

**Comment by PyDataBlog** [comment]
#13 fixes this bug. I will update the package today.

**Comment by PyDataBlog** [comment]
@bicced , the fix has been released now in `0.1.8`
```py
# import the Historical class
from coinsta.core import Historical
from datetime import date

# specify dates considered
start = date(2018, 3, 1)

# get data
data = Historical('yoyow', start=start).get_data()
data2 = Historical('gxchain', start=start).get_data()

# check data heads
print(data.head())
print(data2.head())
```

Please update and close the issue if it works for you.

**Comment by PyDataBlog** [comment]
Closing issue since the bug has been fixed and the OP hasn't responded. Re-open if otherwise.
